src/core/stdcpp/exception.d

```diff
@@ -112,5 +112,13 @@ class bad_exception : exception
 {
 @nogc:
     ///
-    this(const(char)* message = "bad exception") { super(message); }
+    this(const(char)* message = "bad exception") nothrow { super(message); }
+
+    version (GenericBaseException)
+    {
+        ///
+        ~this() nothrow;
+        ///
+        override const(char)* what() const nothrow;
```

Comment:
Required to have the vtable generated on the D side feature the right override in that vtable slot.